### DISCUSSION TITLE
What is too many lists?

### DISCUSSION THREAD

**Discussion by katastrophe92** [celzero/rethink-app#914] [Q&A]
Hello Rethink community!

Everything works good and I had not that many times when I needed to unblock/block manually something. But also idk if I have too many lists and if it can cause problems or eats resources. If someone can tell me if my lists are ok or not would be great. Thanks!!

![signal-2023-05-21-203921_004](https://github.com/celzero/rethink-app/assets/103950507/d26dae22-7429-4615-93d2-8432053f905a)
![signal-2023-05-21-203921_003](https://github.com/celzero/rethink-app/assets/103950507/1efa9357-222b-4f4e-b478-9a6b23cd2980)
![signal-2023-05-21-203921_002](https://github.com/celzero/rethink-app/assets/103950507/90036c36-0d6a-4e17-a244-8f816a467041)

**Comment by MasterKia** [comment]
No one has a crystall ball to guess what lists you have enabled.

**Comment by katastrophe92** [comment]
@MasterKia it did not uploaded the pics I choose the first time. Hope now is visible

**Reply by ignoramous** [reply]
You can instead share the URL?
- If you're using *on-device blocklists*, tap on `Copy as RDNS+ URL` at the bottom of the bottom sheet that comes up when you tap on `On-device blocklists`, and paste it here.
- If you're using remote blocklists (via Rethink DNS), then tap on "`i`" icon next to the selected RDNS entry, then tap on the `Copy` button in the dialog box that shows up. Paste the URL here.
- The URL would look like this: `https://sky.rethinkdns.com/1:YAYBACABEDAgAA==`
 
> If someone can tell me if my lists are ok or not would be great.

I usually avoid the ones marked `red`.

> But also idk if I have too many lists and if it can cause problems or eats resources.

Nah, it won't. Rethink can handle ALL 200 lists (and 14 million odd entries in them) without breaking sweat.

**Comment by katastrophe92** [comment]
@ignoramous Thank you for your time explaining ☺
https://rethinkdns.com/configure?tstamp=1686261361229&s=added#1:4B8kAAIAAAEQDRASABggAM6i
